### DISCUSSION TITLE
Forem cannot start in vscode dev container

### DISCUSSION THREAD

**Discussion by onlinegrave** [forem/forem#21378] [Q&A]
/workspaces/forem (tech/naming_predicate_definition) $ bin/startup                                       
== STARTING UP ==
js      | Running...
web     | Running...
sidekiq | Running...
js      | [watch] build finished, watching for changes...
web     | /workspaces/forem/vendor/bundle/ruby/3.3.0/gems/ffi-compiler-1.0.1/lib/ffi-compiler/loader.rb:21:in `find': cannot find 'http-parser-ext' library (LoadError)
web     |       from /workspaces/forem/vendor/bundle/ruby/3.3.0/gems/http-parser-1.2.3/lib/http-parser/ext.rb:8:in `<module:HttpParser>'

**Comment by Shubham00043** [comment]
It looks like the `http-parser-ext` native dependency may not have been built correctly inside the dev container.

You can try rebuilding the environment with:

```bash id="n8v2m1"
bundle install
```

or remove and reinstall dependencies:

```bash id="x5p7q3"
rm -rf vendor/bundle
bundle install
```

Also try rebuilding the VS Code dev container completely and ensure Docker is running properly before starting Forem again.

Sometimes native Ruby extensions fail during container setup and require a fresh rebuild.
